demos/classical_ct_headneck_affine/README.md

```diff
@@ -1,7 +1,7 @@
 # Classical Affine Registration for Head-and-Neck CT Images
 
 > **Note**: Please read the
-> [DeepReg Demo Disclaimer](introduction.html#demo-disclaimer).
+> [DeepReg Demo Disclaimer](https://deepreg.readthedocs.io/en/325-improve-contributing-pages/demo/introduction.html#demo-disclaimer).
```

Comment:
why this change? there should be no 404 without this change

Reply:
> 
> 
> why this change? there should be no 404 without this change

Did you click on that? Still not works for me.

Reply:
how you tested this and found it buggy? to test the docs, either you do it locally, either we activate the branch on rtd